cmd/data-crawler.go

```diff
@@ -95,11 +95,14 @@ func runDataCrawler(ctx context.Context, objAPI ObjectLayer) {
 		}
 	}
 
+	crawlerTimer := time.NewTimer(dataCrawlStartDelay)
```

Comment:
No, this will only fire once, then never again.

Using a Ticker is tricky, since it will fire irregardsless of the time it takes.